lib/Runtime/Library/JavascriptFunction.cpp

```diff
@@ -697,7 +696,7 @@ namespace Js
                 UnexpectedExceptionHandling_fatal_error(&exceptionInfo);
             }
```

Comment:
This is one example the dummy `__try/__finally` approach brings confusion. This finally block is meaningless and not applicable. I think the before-change code is clearer.